### ISSUE TITLE
fatal: not a git repository (or any of the parent directories): .git

### ISSUE THREAD

**Issue by Gys** [chriswalz/bit#99]
**Describe the bug**

I am in a directory with a .git folder but get:
`fatal: not a git repository (or any of the parent directories): .git`

**To Reproduce**

1. MacOS Big Sur 11.2.1, iTerm2 3.4.4, Bash 3.2.57(1)-release
2. `brew install bit-git`
3. `bit complete` answered `y`
4. `bit` <return> gives the error
5. `ls .git` gives contents of git folder

**Comment by karx1** [comment]
You need to be inside a git repository directory for git (and by extention bit) can do anything.

**Comment by Gys** [comment]
@karx1: I am. Is that not clear from my description? I mention twice that I am doing all this inside a folder with a .git subfolder, so effectively inside a repository.

**Comment by imranansari** [comment]
Updating git worked for me

```
❯ bit --version
bit version v1.0.1
git version 2.30.1
```

**Comment by mdodell** [comment]
This also happened to me. Currently on macOS Catalina, 10.15.7. I tried installing with both the shell script and with Homebrew.

My specs:
```
> bit --version
bit version v1.0.1
git version 2.17.0
```

I am currently in a Git folder:
```
> ls .git
HEAD		config		hooks		objects
branches	description	info		refs
```

**Comment by chriswalz** [comment]
This issue should now be fixed in version v1.0.2. Please let me know if you're stilling having problems. Also, consider donating if you're enjoying using Bit :)
